CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## [0.1.4] - 2019-08-02
```

Comment:
Could you change the release to 0.1.5? There's already a 0.1.4 tag on the repo.

Reply:
Oh, didn't see that. Will also add something into the change log for that version then.